### PR TITLE
webpack-loader not running on all code

### PR DESCRIPTION
**Pull request by oBusk**
**Summary**

I was running into issues where the linaria `webpack-loader` would not apply on my code written in typescript. Managed to find that typescript outputs tagged templates with a space after the function
```
css `
    background: black;
`;
```

This meant that the regex did not match and code did not run.

**Test plan**

After modifying code, linaria was correctly applied to typescript output.

### CODE REVIEW

File: src/tools/webpack-loader.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function shouldRunLinaria(source: string) {
   return (
     (/import .+ from ['"]linaria['"]/g.test(source) ||
       /require\(['"]linaria['"]\)/g.test(source)) &&
-    /css(\.named)?`/g.test(source)
+    /css(\.named)?\s?`/g.test(source)
```
Comment (thymikee) [inline src/tools/webpack-loader.js]: Actually we could adjust it to `\s+` because having more whitespace is also semantically correct

Comment (oBusk) [reply src/tools/webpack-loader.js]: Then surely `*` would be appropiate (Zero or more) rather than `+` (one or more)?

Comment (thymikee) [reply src/tools/webpack-loader.js]: Oh yea, this is what I meant 😄

**Comment by oBusk** [comment]
@zamotany 👌 Done!

**Pull request merged**